priv/repo/migrations/20190413180607_create_auctions.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Renaissance.Repo.Migrations.CreateAuctions do
+  use Ecto.Migration
+
+  def change do
+    create table(:auctions) do
+      add :title, :string
```

Comment:
Probably best to also add `null: false` to the database schema if these are required.

Reply:
I'd only do that for the super core fields, not the aux ones